Sources/SQLiteKit/Docs.docc/Documentation.md

```diff
@@ -0,0 +1,22 @@
+# ``SQLiteKit``
+
+@Metadata {
+    @TitleHeading(Package)
+}
+
+SQLiteKit is a library providing an SQLKit driver for SQLiteNIO.
+
+## Overview
+
+This package provides the "foundational" level of support for using [Fluent] with SQLite by implementing the requirements of an [SQLKit] driver. It is responsible for:
+
+- Managing the underlying SQLite library ([SQLiteNIO]),
+- Providing a two-way bridge between SQLiteNIO and SQLKit's generic data and metadata formats,
+- Presenting an interface for establishing, managing, and interacting with database connections.
+
+> Note: The FluentKit driver for SQLite is provided by the [FluentSQLiteDriver] package.
+
+[SQLKit]: https://swiftpackageindex.com/vapor/sql-kit
```

Comment:
Why are we pointing to the SPI out of interest?

Reply:
If I remember my original reasoning correctly, it was because the SPI's front page for a given package is more convenient in several ways than GitHub's.

Reply:
Cool I have no objections to be clear